dapps/assets/dbcp.json

```diff
@@ -6,6 +6,7 @@
         "bootstrap.vue.libs": "^2.1.0",
         "evancore.vue.libs": "^1.9.0",
         "digitaltwin.lib": "^1.8.0",
+        "digital-twin-detail": "^1.0.0",
```

Comment:
If we don't load something from this DApp, we can simply drop it from the DBCP, because it's get preloaded without using it. If we are removing the `digitaltwin.lib` and switching to the `digital-twin-detail` , this can be added again.
```suggestion
```